breadth_first_traversal/BreadthFirstTraversal.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.*;
 
-public class breadthFirstTraversal {
+public class BreadthFirstTraversal {
```

Comment:
Better change the names to `BreadthFirstSearch` instead of `BreadthFirstTraversal`.